settings.gradle

```diff
@@ -97,6 +97,7 @@ List projects = [
   'test:fixtures:minio-fixture',
   'test:fixtures:old-elasticsearch',
   'test:fixtures:s3-fixture',
+  'test:fixtures:testcontainer-utils',
```

Comment:
have a project where we keep general test container utils. e.g. common ThreadLeakFilter or base classes